docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -151,6 +151,52 @@ to perform the remaining query processing on a smaller amount of data.
 The specifics for the supported pushdown of table joins varies for each data
 source, and therefore for each connector.
 
+However, there are some generic conditions that must be met in order for a join
+to be pushed down:
+
+* join pushdown must be enabled in a catalog
+* all predicates that are part of the join must be possible to be pushed down
+* the tables in the join must be from the same catalog
+
+You can verify if pushdown for a specific join is performed by looking at the
+:doc:`EXPLAIN plan </sql/explain>` of the query. The explain plan does not
+show a ``Join`` operator, if the join is pushed down to the data source by the
+connector.
+
+Prepend ``EXPLAIN`` to a query to access the explain plan::
```

Comment:
Let's remove this sentence, since it's repeating what is in the EXPLAIN documentation linked above, and in the above sections. It certainly doesn't warrant a separate paragraph, and we don't need to repeat this for every section.